Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml

```diff
@@ -0,0 +1,563 @@
+              suffix: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 910,
+          "y": 45
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "47":
+    id: "47"
+    taskid: ce97d194-4dca-4f9c-8aaf-7c54ab40e966
+    type: regular
+    task:
+      id: ce97d194-4dca-4f9c-8aaf-7c54ab40e966
+      version: -1
+      name: Get Actor CommandLine and CGO CommandLine prevalence
+      description: Get the prevalence of a process_command_line, identified by process_command_line.
+      script: '|||core-get-cmd-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "61"
+    scriptarguments:
+      process_command_line:
+        complex:
+          root: alert
+          accessor: cgocmd
+          transformers:
+          - operator: AppendIfNotEmpty
+            args:
+              item:
+                value:
+                  simple: alert.osparentcmd
+                iscontext: true
+              raw: {}
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 650,
+          "y": -300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "49":
+    id: "49"
+    taskid: e0e01cdc-0f66-414b-8558-24155f2650e7
+    type: regular
+    task:
+      id: e0e01cdc-0f66-414b-8558-24155f2650e7
+      version: -1
+      name: Get Actor Process and CGO Process prevalence
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "61"
+    scriptarguments:
+      process_name:
+        complex:
+          root: alert
+          accessor: osparentname
+          transformers:
+          - operator: AppendIfNotEmpty
+            args:
+              item:
+                value:
+                  simple: alert.cgoname
+                iscontext: true
+              raw: {}
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": -300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "53":
+    id: "53"
+    taskid: 76b1f4c5-07a8-45d9-8f47-a2e6e65215cc
+    type: condition
+    task:
+      id: 76b1f4c5-07a8-45d9-8f47-a2e6e65215cc
+      version: -1
+      name: Found any related alerts?
```

Comment:
change to Found related alerts requiring causality termination?

Reply:
done